Explorer/Assets/DCL/LOD/Systems/UpdateSceneLODInfoSystem.cs

```diff
@@ -46,13 +46,16 @@ public partial class UpdateSceneLODInfoSystem : BaseUnityLoopSystem
         protected override void Update(float t)
         {
             UpdateLODLevelQuery(World);
+
             //InstantiateCurrentLODQuery(World);
         }
 
         [Query]
         [None(typeof(DeleteEntityIntention))]
         private void UpdateLODLevel(ref SceneLODInfo sceneLODInfo, ref PartitionComponent partitionComponent, SceneDefinitionComponent sceneDefinitionComponent)
         {
+            if (sceneDefinitionComponent.IsPortableExperience) return;
```

Comment:
I think it would be better to add a new component, something like `struct PortableExperience`, so you can filter this queries by it: `[None(typeof(PortableExperience))]`

Reply:
Yeah, that's a great option! I think that when we add the SceneDefinitionComponent we can add that other component as well, nice!